grid/social-media/followers/profile.supr

```diff
@@ -0,0 +1,95 @@
+name = "social-media/followers"
+version = "1.0.0"
+
+"""
+Get Followers
+Retrieve paginated list of social-media profile followers.
+"""
+usecase GetFollowers safe {
+  input {
+    """
+    Profile ID
+    Identification of profile to retrieve posts from; may be optional with some providers.
+    """
+    profileId string
```

Comment:
Similar to lookups, we should allow fetching that with username, but that can be added later.